src/YieldDisburser.sol

```diff
@@ -135,6 +135,25 @@ contract YieldDisburser is OwnableUpgradeable {
         __Ownable_init(msg.sender);
     }
 
+    /**
+     * @notice Returns the current distribution of voting power for projects
+     * @return address[] The current breadchain projects
+     * @return uint256[] The current distribution of voting power for projects
+     */
+    function getCurrentVotingDistribution() public view returns (address[] memory, uint256[] memory) {
+        return (projects, projectDistributions);
+    }
+
+    /**
+     * @notice Return the current voting power of a user
```

Comment:
should include a `@param _account` comment here

Reply:
amended with e0cfac4